sdk/communication/communication-tiering/review/communication-tiering.api.md

```diff
@@ -0,0 +1,204 @@
+## API Report File for "@azure-tools/communication-tiering"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference lib="esnext.asynciterable" />
+
+import { CommonClientOptions } from '@azure/core-client';
+import * as coreClient from '@azure/core-client';
+import { KeyCredential } from '@azure/core-auth';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export interface AcsTier {
+    capabilities?: AcsTierCapabilities;
+    tierId?: string;
+    tierType?: string;
+}
+
+// @public
+export interface AcsTierCapabilities {
+    calling?: AcsTierCapabilitiesCalling;
+    phoneNumberPurchase?: PhoneNumberPurchase[];
+    sms?: AcsTierCapabilitiesSms;
+    trialPhoneNumberPurchase?: TrialPhoneNumberPurchase[];
+    trialPhoneNumberUsage?: TrialPhoneNumberUsage[];
+}
+
+// @public
+export interface AcsTierCapabilitiesCalling {
+    inbound?: CallingInbound[];
+    outbound?: CallingOutbound[];
+}
+
+// @public (undocumented)
+export const AcsTierCapabilitiesCallingMapper: coreClient.CompositeMapper;
+
+// @public (undocumented)
+export const AcsTierCapabilitiesMappers: coreClient.CompositeMapper;
+
+// @public
+export interface AcsTierCapabilitiesSms {
+    inbound?: SmsInbound[];
+    outbound?: SmsOutbound[];
+}
+
+// @public (undocumented)
+export const AcsTierCapabilitiesSMSMapper: coreClient.CompositeMapper;
+
+// @public (undocumented)
+export const AcsTierMapper: coreClient.CompositeMapper;
+
+// @public
+export interface AssetDetails {
+    aadTenantAcquiredCount?: number;
+    numberType?: string[];
+    resourceAcquiredCount?: number;
+    subscriptionAcquiredCount?: number;
+    tierInfo?: AssetDetailsTierInfo;
+}
+
+// @public (undocumented)
+export const AssetDetailsMapper: coreClient.CompositeMapper;
```

Comment:
The mappers shouldn't be in the public api surface. Could you check why they are exported?

Reply:
Ok fixed.